system/jlib/jevent.cpp

```diff
@@ -91,28 +91,38 @@ struct EventInformation
     bool isMeta;                // Is this meta data (see description below) rather than an event
     EventType pairedEvent;      // could be used to automaticaly link start and finish events?
     EventContext context;       // grouping related events
+    EventAttr attributes[EvAttrMax];     // EvAttrNone-terminated array of attributes
```

Comment:
Try using:
    std::initializer_list<EventAttr> attributes;

It will be cleaner - avoiding the need for the terminator, and allow the use in for() loops/